gabarit/template_num/num_project/package_name/models_training/classifiers/model_classifier.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, level_save: str = 'HIGH', multi_label: bool = False, **kwargs
         self.level_save = level_save
 
     @utils.trained_needed
-    def predict_with_proba(self, x_test: pd.DataFrame) -> Tuple[np.ndarray, np.ndarray]:
+    def predict_with_proba(self, x_test: pd.DataFrame, **kwargs) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
In this method, the kwargs are not passed on to the self.predict below. Is it correct?